### PR TITLE
Feature/docker arm versions

### PR DESCRIPTION
**Pull request by aseefahmed**
## Description

Right now we only build our docker images for a single platform. We should also build ARM versions of our docker images for use with M1 Macbooks and AWS ARM instances.

## Motivation and Context

Add the additional parameters to our docker builds to build and push images for ARM platforms as well as our current builds.

## How Has This Been Tested?

PR with images building for ARM as well as x86

### CODE REVIEW

**Comment by aseefahmed** [comment]
@jonathangreen to review

**Comment by jonathangreen** [comment]
This builds and pushes an arm version of the image, but it doesn't do any tests on it. Can you also run the tests scripts on the arm versions of the image?

**Review by jonathangreen — CHANGES_REQUESTED** [review]
This builds and pushes an arm version of the image, but it doesn't do any tests on it. Can you also run the tests scripts on the arm versions of the image?

**Comment by aseefahmed** [comment]
> This builds and pushes an arm version of the image, but it doesn't do any tests on it. Can you also run the tests scripts on the arm versions of the image?

will have a try

**Comment by aseefahmed** [comment]
@jonathangreen to review

File: .github/workflows/test-build.yml [hunk 1]
```diff
@@ -98,32 +101,85 @@ jobs:
       - name: Create version file
         run: git rev-parse --short HEAD > .version
 
-      - name: Build circ-scripts image
+      - name: Build circ-scripts image (linux/amd64)
```
Comment (jonathangreen) [inline .github/workflows/test-build.yml:104]: Instead of copy pasting these steps you should be able to use a build matrix here.

Comment (jonathangreen) [reply .github/workflows/test-build.yml:104]:
```yaml
    strategy:
      fail-fast: false
      matrix:
        platform: ["linux/amd64", "linux/arm64"]
```

**Review by jonathangreen — CHANGES_REQUESTED** [review]

**Comment by aseefahmed** [comment]
@jonathangreen to review

**Review by jonathangreen — APPROVED** [review]

**Pull request merged**